pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -295,17 +295,24 @@ func (c *component) containers() []corev1.Container {
 		}
 
 		if c.config.WAFEnabled {
-			commandArgs = append(commandArgs, "--rules", "/etc/modsecurity-ruleset")
+			commandArgs = append(
+				commandArgs,
+				"--waf-enabled",
```

Comment:
I think we need to pass --dial as /var/run/nodeagent/socket command line argument to Dikastes

Reply:
It's already added to line# 288